src/averageAges.js

```diff
@@ -37,7 +48,23 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const FEMALE_SIGN = 'f';
+
+  let females = people.filter(({ sex }) => sex === FEMALE_SIGN);
+
+  if (withChildren) {
+    females = females.filter(({ name }) => {
+      return people.some(({ mother }) => mother === name);
```

Comment:
If function returns result without any calculations, you can simplify it and avoid using `return`

Reply:
Sure. The only reason I did it is to make line shorter, because

females = females.filter(({ name }) => people.some(({ mother }) => mother === name));

is longer than 80 chars.